State_manager_code/StateManager.py

```diff
@@ -99,7 +102,7 @@ def __init__(self):
             States.WaitForReturn: 10,  # 10 seconds is a default value. It is replaces by the SetTimeOut functions.
             States.TrialCompleted: 0,
             States.TrialAbort: None,
-            States.DecisionAbort: 0,}
+            States.DecisionAbort: None,}
```

Comment:
Also in the TimeOutState table you need to change
States.DecisionAbort: States.TrialStarted, to
States.DecisionAbort: None,

Reply:
Anushka, come to think about it. There are more issues in the StateManger tables. I think it does not work as you expect.
Let's talk about it before you merge the branch.